config.js

```diff
@@ -487,12 +488,14 @@ function testDroppedArgv(argv) {
 
 function ensureSupportedOgspvAI(ogspv, ogsPvAIs) {
     // being case tolerant
-    const upperCaseOgsPv = ogspv.toUpperCase();
-    const upperCaseAIs   = ogsPvAIs.map(e => e.toUpperCase());
+    if(ogspv) {
```

Comment:
it is a good idea to add the `if`, but i think we would want to make the `if` outside of the function, and run the function only if the arg is used, what do you think @Dorus 

```Js
if (argv.ogspv) ensureSupportedOgspvAI(ogspv, ogsPvAIs);
```

Reply:
fixed

Reply:
it's not just a good idea, it crashed the damn program when I used it :o